pkg/resourceinterpreter/defaultinterpreter/default.go

```diff
@@ -138,3 +157,29 @@ func (e *DefaultInterpreter) InterpretHealth(object *unstructured.Unstructured)
 
 	return false, fmt.Errorf("default %s interpreter for %q not found", configv1alpha1.InterpreterOperationInterpretHealth, object.GroupVersionKind())
 }
+
+func (e *DefaultInterpreter) checkScaleSubResourceExist(kind schema.GroupVersionKind) bool {
+	mapping, err := e.restMapper.RESTMapping(kind.GroupKind(), kind.Version)
+	if err != nil {
+		klog.Errorf("Failed to get RESTMapping of resource %v: %v", kind, err)
+		return false
+	}
+
+	gvr := mapping.GroupVersionKind.GroupVersion().WithResource(mapping.Resource.Resource)
+	if _, err = e.scaleKindResolver.ScaleForResource(gvr); err != nil {
+		klog.Warningf("Cannot get scale subresource for %v: %v", gvr, err)
```

Comment:
Does this mean this `warning` log will be dumped very very often? Have you checked the logs when testing it?